whatrecord/db.py

```diff
@@ -777,6 +777,33 @@ def add_or_update_record(self, record: RecordInstance):
         else:
             existing_record.update(record)
 
+    def shallow_copy(self) -> Database:
+        """A shallow copy of the database with new dictionaries/lists."""
```

Comment:
`copy.copy()` didn't work here?

Reply:
To my understanding, `copy.copy` is a [_very shallow_ copy](https://docs.python.org/3/library/copy.html) - ("A shallow copy constructs a new compound object and then (to the extent possible) inserts references into it to the objects found in the original.").

What's implemented here is a _slightly less shallow_ copy.

In order of depth:
1. `copy.copy(db)` - "new Database instance, same `list` instances"
2. `db.shallow_copy()` - "new Database instance, new `list` instances with same contents"
3. `copy.deepcopy(db)` - "new Database instance, new `list` instances, new copies of contents"